### PR TITLE
test: python multi thread

### PR DESCRIPTION
**Pull request by t-h2o**
# test with many thread

- split function main
- launch a lot of request

## Resource

* Intro: https://realpython.com/intro-to-python-threading/#starting-a-thread
* all documentations: https://docs.python.org/3/library/threading.html

### CODE REVIEW

File: python/script.py [hunk 1]
```diff
@@ -58,6 +60,20 @@ def main ():
...
+    all_treads = []
+
+    for i in range(TREADS_NUMBER):
+        all_treads.append(threading.Thread(target=my_thread, args=[1,]))
+    for i in range(TREADS_NUMBER):
+        all_treads[i].run()
+
+def main ():
+    multi_request()
+
```
Comment (0xCAF3D0OD) [inline python/script.py:76]: For wich purpose this files exist, why multithreading ?

Comment (t-h2o) [reply python/script.py:76]: * The purpose of the file is explained here #54 .

* Why multi threading, to simulate many client trying to connect into our web server

File: python/script.py [hunk 2]
```diff
@@ -58,6 +60,20 @@ def main ():
...
     request_get("http://webserv.com:8080/favicon.ico", 404)
 
+def my_thread (name):
+    basic_status_code_test()
+
+def multi_request ():
+    all_treads = []
+
+    for i in range(TREADS_NUMBER):
+        all_treads.append(threading.Thread(target=my_thread, args=[1,]))
```
Comment (RobinBurri) [inline python/script.py:70]: Tu peux expliquer cette ligne (surtout args=[1,])?

Comment (t-h2o) [reply python/script.py:70]: A thread take as argument a reference on a function (`target`), this function can have 0, 1 or many arguments,
the goal of `args` is to give argument to the function. For example `[1,]` is identical to `[1]` and that means, the arguments is 1.

I change `1` by `i` to give the thread number to the thread.

Comment (RobinBurri) [reply python/script.py]: Nickel, merci.

**Review by RobinBurri — APPROVED** [review]
C'est beau tout ca.

**Review by 0xCAF3D0OD — APPROVED** [review]
OK

**Pull request merged**